cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts

```diff
@@ -71,8 +71,12 @@ describe('Test Case Expected Measure Group population values based on initial me
         cy.get(TestCasesPage.testCasePopulationValuesTable).should('be.visible')
         cy.get(TestCasesPage.testCasePopulationValues).should('contain.text', 'PopulationExpectedActual')
 
-        //validate proper Population Value check boxes, based on value of score
-        Utilities.validateTCPopValueCheckBoxes((measureScoringArray[3].valueOf()).toString())
+        cy.get(TestCasesPage.testCaseIPPCheckBox).should('be.visible')
+        cy.get(TestCasesPage.testCaseIPPCheckBox).check().should('be.checked')
```

Comment:
What does the .check() do? Just curious

Reply:
The '.check()' checks the check box. In these two lines, I am 1.) making sure the checkbox for the Initial Population check box is visible; 2.) checking the check box; 3.) verifying that after checking the check box that it is, in fact, checked